submissions/Roophee/JS - DOM/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://cdn.jsdelivr.net/npm/boxicons@2.0.5/css/boxicons.min.css' rel='stylesheet'>
+    <title>Short stories</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="burger" tabindex="1">
+        </div>
+        <heeader class="header">
+            <div class="label">
+                <span>
+                    Your short story
+                </span>
+            </div>
+        </heeader>
+        <div class="content__container">
+            <aside class="aside">
```

Comment:
The content inside HTML tag `aside` has no semantic connection with content inside HTML tag `main`. In your case, you encapsulated navigation and semantically content has no connection with it.
Replace this tag with `div` or remove this wrapper)

Reply:
Done.